gearcmd/worker_test.go

```diff
@@ -24,6 +26,24 @@ func getSuccessResponseWithConfig(payload string, config TaskConfig, t *testing.
 	return string(mockJob.OutData())
 }
 
+func testRetryOnFailure(retryCount int) ([]byte, error) {
+	// Get a temp file for the script to hold its state.
+	file, err := ioutil.TempFile("", "temp")
+	if err != nil {
+		return nil, fmt.Errorf("Could not create temporary file: %s", err.Error())
+	}
+	filename := file.Name()
+	defer os.Remove(filename)
+	defer file.Close()
+	mockJob := mock.CreateMockJob(filename)
```

Comment:
Ah, yeah!  OK, that's what I originally had in mind but I misread what you had and thought you were using an environment variable.  :+1: